viadot/task_utils.py

```diff
@@ -304,6 +305,57 @@ def cleanup_validation_clutter(expectations_path):
     shutil.rmtree(ge_project_path)
 
 
+@task
+def generate_table_dtypes(
+    table_name: str = None,
+    db_name: str = None,
+    reserve: float = 1.4,
+    config_key: str = None,
+    only_dict: bool = True,
+    credentials: str = None,
+    schema: str = None,
+) -> dict:
+    """Functon that automaticy generate dtypes dict from SQL table.
+
+    Args:
+        table_name (str): Table name. Defaults to None.
+        db_name (str): Data base name. Defaults to None.
+        reserve (str): How many signs add to varchar, percentage of value. Defaults to 1.4.
+        config_key (str): The key inside local config containing the config.
+        only_dict (bool): choose to generate dictionary or whole dataframe. Defaults to True.
+
+    Returns:
+        Dictionary
+    """
+    from viadot.sources import AzureSQL
+
+    sql = AzureSQL(config_key=config_key, credentials=credentials)
+    if db_name:
+        sql.credentials["db_name"] = db_name
+
+    query_admin = f"""select COLUMN_NAME, DATA_TYPE, CHARACTER_MAXIMUM_LENGTH, 
+        NUMERIC_PRECISION, DATETIME_PRECISION, 
+        IS_NULLABLE 
+        from INFORMATION_SCHEMA.COLUMNS
+        where TABLE_NAME='{table_name}' and TABLE_SCHEMA='{schema}'
+        order by CHARACTER_MAXIMUM_LENGTH desc"""
+
+    if sql.con:
```

Comment:
Is better to write in this way: `if sql_conn == True` if  sql.conn means `True` or `False`